chart/values.yaml

```diff
@@ -327,6 +327,12 @@ csi:
     tolerations: []
     # -- Set PriorityClass, overrides global
     priorityClassName: ""
+    initContainers:
+      enabled: false
```

Comment:
Why do we need this switch?

Reply:
Not enabled for mayastor chart itself, the umbrella should enable it.